cli/napi/node_api.rs

```diff
@@ -0,0 +1,693 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+#![deny(unsafe_op_in_unsafe_fn)]
```

Comment:
According to https://doc.rust-lang.org/edition-guide/rust-2024/unsafe-op-in-unsafe-fn.html this lint is not yet released. Are you using nightly Rust? Do we need to use this lint?

Reply:
the lint works fine, that page is talking about enabling it by default.